initial/src/main/java/com/example/resources/PricingResource.java

```diff
@@ -0,0 +1,25 @@
+package com.example.resources;
+
+import com.example.services.PricingService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PricingResource {
```

Comment:
This resource is not that useful. Pricing won't be standalone, instead it will always be associated with the course data. 

Maybe in the future, when the system gets more complex, we will expose pricing info to the client and this can be useful at that time. But let's discard this for now.

Reply:
ah yes, I think we had chatted about this before. deleted.